src/finding/mod.rs

```diff
@@ -267,18 +268,54 @@ impl<'w> FindingBuilder<'w> {
     }
 
     pub(crate) fn build(self, workflow: &'w Workflow) -> Result<Finding<'w>> {
+        let locations = self
+            .locations
+            .iter()
+            .map(|l| l.clone().concretize(workflow))
+            .collect::<Result<Vec<_>>>()?;
+
+        let should_ignore = self.ignored_from_inlined_comment(workflow, &locations, self.ident);
+
         Ok(Finding {
             ident: self.ident,
             desc: self.desc,
             determinations: Determinations {
                 confidence: self.confidence,
                 severity: self.severity,
             },
-            locations: self
-                .locations
-                .into_iter()
-                .map(|l| l.concretize(workflow))
-                .collect::<Result<Vec<_>>>()?,
+            locations,
+            ignored_from_inlined_comment: should_ignore,
         })
     }
+
+    fn ignored_from_inlined_comment(
+        &self,
+        workflow: &Workflow,
+        locations: &[Location],
+        id: &str,
+    ) -> bool {
+        let document_lines = &workflow.contents.lines().collect::<Vec<_>>();
+        let line_ranges = locations
+            .iter()
+            .map(|l| {
+                (
+                    l.concrete.location.start_point.row,
+                    l.concrete.location.end_point.row,
+                )
+            })
+            .collect::<Vec<_>>();
+
+        let inlined_ignore = format!("zizmor: ignore[{}]", id);
```

Comment:
Flagging for discussion: we can think about this more in a follow-up, but I think we want a bit more syntax here to enable multiple ignores.

For example:

1. Ignore everything: `# zizmor: ignore[*]`
2. Ignore two specific rules: `# zizmor: ignore[rulea,ruleb]`

We can start with the current behavior though, to avoid making this PR too big :slightly_smiling_face:

Reply:
https://github.com/woodruffw/zizmor/issues/189